### PR TITLE
fix : adds the LNClientException dependency

### PR DESCRIPTION
**Pull request by Harshit933**
This commit adds the LNClientException class for all the packages by importing it from the rpc package.

### CODE REVIEW

File: packages/lnlambda/pubspec.yaml [hunk 1]
```diff
@@ -16,7 +17,8 @@ dev_dependencies:
   test: ^1.16.0
   build_runner: ^2.0.0
   json_serializable: ^6.2.0
-  clightning_rpc: ^0.0.2-beta.1
+  clightning_rpc:
+    path: ../rpc
```
Comment (vincenzopalazzo) [inline packages/lnlambda/pubspec.yaml:21]: if you use the override dependencies here you can remove the `publish_to: none`?

File: packages/rpc/pubspec.yaml [hunk 2]
```diff
@@ -13,7 +13,8 @@ dev_dependencies:
 dependencies:
   json_rpc_2: ^3.0.1
   jsonrpc2: ^3.0.1
-  cln_common: ^0.0.1-beta.1
+  cln_common:
+    path: ../cln_common
```
Comment (vincenzopalazzo) [inline packages/rpc/pubspec.yaml:17]: same question here

**Review by vincenzopalazzo — CHANGES_REQUESTED** [review]

File: packages/lnlambda/pubspec.yaml (hunk 1 above)
Comment (Harshit933) [reply packages/lnlambda/pubspec.yaml:21]: I am getting this warning in this case `Publishable packages can't have 'path' dependencies.`

**Review by vincenzopalazzo — APPROVED** [review]
LGTM

**Pull request merged**